libs/api/domains/ship-registry/src/lib/dto/ship-search.input.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType('ShipRegistryShipSearchInput')
+export class ShipSearchInput {
+  @Field(() => String)
+  qs!: string
```

Comment:
Since the xroad endpoint calls it qs I for whatever reason just used the same name here, I could've used something else but for the sake of time I'd like to keep it like this if that's alright